target/riscv/gdbstub.c

```diff
@@ -228,6 +228,103 @@ static int riscv_gen_dynamic_csr_xml(CPUState *cs, int base_reg)
     return CSR_TABLE_SIZE;
 }
 
+#if defined(TARGET_CHERI)
+static struct SCR {
+    int index;
+    const char *name;
+    bool code;
+} scrs[] = {
```

Comment:
This duplicates scr_info in cheri_op_helper.c a bit.  However, using all-lowercase names is more consistent with other registers exported via GDB.